packages/sync/src/conflicts/base/BaseStateStore.ts

```diff
@@ -0,0 +1,41 @@
+import { BaseStateProvider } from "./BaseStateProvider";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import { Store } from "idb-localstorage";
+
+/**
+ * Store that provides access to base state along with local persistence.
+ */
+export class BaseStateStore implements BaseStateProvider {
+  public baseState: any = {};
+  private storage: Store;
+
+  constructor() {
+      this.storage = new Store("base-store", "base-data");
```

Comment:
I think it would make sense to pass the storage implementation ("idb-localstorage") into the constructor here. It means we will need to define some basic interface for the storage but it will make this entire file really easy to unit test

Reply:
Yeah. We need to abstract this. Another TODO :)